users/README.md

```diff
@@ -22,6 +22,7 @@
   'isMember': boolean,
   'userType': string,
   'tokens': {},
-  'status': string
+  'status': string,
+  'roles': {}
```

Comment:
Should we add keys available in roles also?

Reply:
@prakashchoudhary07 Yes, as it is also done in others like `external-accounts` and `applicants`. I'm also thinking of adding in an example section like in `external accounts` so that the viewer will get an idea of what kind of data is being stored.